package.json

```diff
@@ -18,19 +18,25 @@
     "eslint-plugin-react": "^7.12.4",
     "husky": "^1.3.1",
     "lint-staged": "^8.1.4",
-    "prettier": "1.16.4",
+    "prettier": "1.17.0",
     "react-select": "^2.0.0-alpha.3",
     "webpack": "^1.14.0"
   },
   "scripts": {
     "dev": "webpack --watch --progress --colors",
-    "build": "webpack -p",
+    "build": "webpack",
```

Comment:
Any reason we're dropping the p flag?

Reply:
Ah it was removed while I was debugging, will add it back in. Nice catch.